angio-frontend/src/modules/user/views/Registration.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+    <CentredLayout>
+        <v-flex xs12 sm6 md5 lg4 xl3>
+            <RegistrationForm
+                    v-on:send-form="sendForm"
+                    v-bind:fetching="fetching"
+                    v-bind:error-messages="errorMessages"
+            ></RegistrationForm>
+        </v-flex>
+    </CentredLayout>
+</template>
+
+<script lang="ts">
+    import {Component, Vue} from 'vue-property-decorator';
+    import CentredLayout from '@/modules/common/components/CentredLayout.vue';
+    import {throttle} from 'helpful-decorators';
+    import RegistrationForm from '@/modules/user/components/RegistrationForm.vue';
+    import {UserRegisterModel} from "@/modules/user/models/user";
```

Comment:
Use single quotes instead of double quotes

Reply:
Done